client/components/chart/style.scss

```diff
@@ -181,6 +238,22 @@
 	}
 
 	li {
+		&.woocommerce-legend__item {
+			button {
+				&:hover {
+					background-color: none;
+				}
+			}
+
+			&.woocommerce-legend__item-checked {
+				button {
+					&:hover {
```

Comment:
This can be written like:
`&.woocommerce-legend__item-checked button:hover {`
Is there any reason to prefer nesting?